assets/js/common.js

```diff
@@ -190,14 +190,111 @@
 				// Set language
 				map.addControl(new MapboxLanguage());
 
-				// Add markers
-				$.each(container.data('markers'), function(index, data) {
-					bounds.extend([data.longitude, data.latitude]);
+				map.on('load', () => {
```

Comment:
Please describe the solution, e.g. source (is this an official one? especially randomizing coordinates).

Reply:
This is the code from the documentation you provided in the task description..
Markers with the same address creating on the same place so it unavailable to click them separately so I've added random offset to their spawn position